pkg/analysis_server/lib/src/services/correction/fix.dart

```diff
@@ -185,6 +186,8 @@ class DartFixKind {
       const FixKind('INSERT_SEMICOLON', 50, "Insert ';'");
   static const LINT_ADD_OVERRIDE =
       const FixKind('LINT_ADD_OVERRIDE', 50, "Add '@override' annotation");
+  static const LINT_REMOVE_INIT_TO_NULL =
+      const FixKind('LINT_REMOVE_INIT_TO_NULL', 50, "Remove initialization to null");
```

Comment:
Please rename to `REMOVE_INITIALIZER` and change the description appropriately. There's nothing about this (or any other) fix that is specific to a lint, and the fix isn't dependent on the form of the initialization expression.